cfgov/unprocessed/js/modules/util/format.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Turns a string into a number.
+ * Assumes each number in the string should be preserved (unlike parseInt)
+ * Assumes the first instance of a decimal point is the intended one.
+ * @param  {string} numberString - A string representing a number.
+ * @returns {number} The assumed numeric value of numberString
+ */
+function convertStringToNumber(numberString) {
+  if (typeof numberString === 'number') {
+    return numberString;
+  } else if (typeof numberString !== 'string') {
+    return 0;
+  }
+
+  let signMaker = 1;
+  const minusPosition = numberString.indexOf(numberString.match('-'));
+  const digitPosition = numberString.indexOf(numberString.match(/\d/));
+  const dashPosition = numberString.indexOf(numberString.match('-'));
+
+  // If a '-' appears before the first digit, we assume numberString is negative
+  if (dashPosition !== -1 && minusPosition < digitPosition) {
+    signMaker = -1;
+  }
+
+  // Strip non-numeric values, maintaining periods
+  numberString = numberString.replace(/[^0-9\.]+/g, '');
+
+  numberString = numberString.replace(/\./g, replaceCommas);
+
+  // Get number value of string, then multiply by signMaker and return
+  return Number(numberString) * signMaker;
```

Comment:
Feels like this should just use `unformatUSD`, no?

Reply:
`unFormatUSD` usurped by `convertStringToNumber` in https://github.com/cfpb/consumerfinance.gov/pull/7400/commits/ca508a84f0be8b450d05c4701c836e00db864d3c